test/test_db.py

```diff
@@ -39,8 +39,8 @@ def test_concurrency():
     service_instance = Mock()
 
     def inject(worker_ctx):
```

Comment:
possibly out of scope, but this looks like it's messing with internals in a way it probably shouldn't